tests/readers/test_batch_utils.py

```diff
@@ -109,3 +115,341 @@ def test_get_num_batches_sparse(sparse_uri):
         assert get_num_batches(batch_size, buffer_bytes, a, attrs=["af8", "au1"]) == 49
         # int(50000 / 16 / 48) == 65 but there are at most ceil(1000 / 16) == 63 batches
         assert get_num_batches(batch_size, buffer_bytes, a, attrs=["af4"]) == 63
+
+
+def test_iter_batches():
```

Comment:
Could we make this more compact?

Reply:
We can reduce `stop_offset` to generate fewer batches (list entries). If you mean making each batch more compact, I guess we can override `Batch.__repr__`.

Reply:
Made test more compact at https://github.com/TileDB-Inc/TileDB-ML/pull/119/commits/5aabe424534f1857efada6e267c6cf4ebf96d438